core/src/main/java/org/bitcoinj/core/TransactionWitness.java

```diff
@@ -64,4 +64,19 @@ public String toString() {
         }
         return Utils.SPACE_JOINER.join(stringPushes);
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+
+        TransactionWitness that = (TransactionWitness) o;
```

Comment:
Nitpick: We usually use the variable name "other" at that point in equals() impls.